pkg/cri/sbserver/images/image_pull_test.go

```diff
@@ -432,7 +431,9 @@ func TestSnapshotterFromPodSandboxConfig(t *testing.T) {
 			}
 			snapshotter, err := cri.snapshotterFromPodSandboxConfig(context.Background(), "test-image", tt.podSandboxConfig)
 			assert.Equal(t, tt.expectSnapshotter, snapshotter)
-			assert.Equal(t, tt.expectErr, err)
+			if tt.expectErr {
+				assert.Error(t, err)
```

Comment:
In a sense, but the truth is we really care about IF there was an error. We weren't returning a constant errors.New style error so this was already kind of awkward. I don't think we're losing much test coverage with this change, but I could be convinced otherwise. expectErr being a bool also makes this match a bunch of other times we use this pattern

Reply:
I mean that's completely fair, quite a bit of our tests fall into the same category though 😅. We could change back to checking the error if you want. It's much easier if it's just a global errors.New style we need to check against though (or if we just care what type the error is, but I don't think this warrants a new error type), I'm not a fan of reconstructing some fmt.Errorf